Resources/Storage/StorageFactory.py

```diff
@@ -35,6 +35,12 @@ def __init__( self, useProxy = False, vo = None ):
         self.vo = result['Value']
       else:
         RuntimeError( "Can not get the current VO context" )
+    self.remotePlugins = []
```

Comment:
Yes I know... I had no idea whether it was required to re-initialize them in getStorages. As it is it doesn't harm and makes the number of pylint warnings decrease... Someone who has time/expertise should check whether they can be removed from getStorages ;-)